packages/shared/src/vault/schemas.ts

```diff
@@ -85,21 +85,18 @@ const nativeCallParamsSchema = ethereumBase.extend({
 
 export type NativeCallParams = z.infer<typeof nativeCallParamsSchema>;
 
-const ccmMetadataSchema = z.object({
-  message: hexString,
-  gasBudget: numericString,
-});
-
 const tokenCallParamsSchema = z.union([
   ethereumBase.extend({
     srcAsset: z.literal(Assets.FLIP),
     destAsset: z.union([z.literal(Assets.USDC), z.literal(Assets.ETH)]),
-    ccmMetadata: ccmMetadataSchema,
```

Comment:
🤔

Reply:
this was published because we don't check that the working directory is clean.